frontend/public/signup.css

```diff
@@ -0,0 +1,104 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background: url('https://rabblerouserteam.files.wordpress.com/2016/08/rabble-rouser-header.jpg') no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  font-family: 'Open Sans', sans-serif;
+}
+
+header {
+  background-color: #EA3B49;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  padding-left: 50px;
+}
+
+header img {
+  content:url('https://i.imgur.com/eecsJFb.png');
+  height: 30px;
+}
+
+main {
+  width: 600px;
+  margin: 20px auto;
+  background-color: white;
+  padding: 40px;
+  overflow: auto;
+}
+
+@media (max-width: 600px) {
+  main {
+    width: 100%;
+  }
+}
+
+h1 {
+  font-weight: bold;
+  text-transform: uppercase;
+  text-align: center;
+  font-size: 24px;
+}
+
+.sub-title {
+  font-family: 'Droid Serif', serif;
+}
+
+fieldset {
+  border: 0;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  display: inline-block;
+}
+
+@media (min-width: 600px) {
+  fieldset.firstName {
+    width: 48%
+  }
+
+  fieldset.lastName {
+    width: 48%;
+    float: right;
+  }
+}
+
+fieldset label {
+  display: block;
+}
+
+input {
+  width: 100%;
+  font-size: 30px;
```

Comment:
👀 Text sizes may be just very slightly in need of tweaking 👀
<img width="606" alt="screen shot 2016-10-11 at 6 35 18 pm" src="https://cloud.githubusercontent.com/assets/10258529/19262260/d1ed366a-8fe1-11e6-8141-6937779e40e4.png">